tests/bazel_tests.bzl

```diff
@@ -0,0 +1,132 @@
+_script_template = """
+ANCHOR=$(readlink -f {ANCHOR})
+BASE=$(dirname $ANCHOR)
+CACHE_FILE=$(realpath {CACHE})
+BAZEL_CACHE=$(dirname $CACHE_FILE)
+cd {PACKAGE}
+bazel {ARGS} {TARGET}
```

Comment:
Most of these template variable expansions should be quoted, but be careful with `ARGS`. Might run into trouble if there's a space above wherever rules_go is checked out.